model/config.go

```diff
@@ -499,6 +499,13 @@ type DataRetentionSettings struct {
 	DeletionJobStartTime  *string
 }
 
+type MessageExportSettings struct {
```

Comment:
Please also add the first three settings to app/diagnostics.go (not `FileLocation`), once added to `config.json`.

Reply:
Fixed in https://github.com/mattermost/mattermost-server/pull/7612/commits/dab3563c873c4ae916c58b439f8ac5094c7000ff

Reply:
@MusikPolice Should we add these to config.json too, or are we holding off on that until the Beta release is confirmed?

Reply:
@jasonblais can you clarify? The settings all have default values, and are automatically added to config.json if they haven't been explicitly set. There's also a System Console page that allows you to set their values if the server is E20 licensed. It's not clear to me that anything else has to be done with respect to configuration.